### DISCUSSION TITLE
How to set default value on ColorPickerDialog

### DISCUSSION THREAD

**Discussion by tonipetrovic** [skydoves/ColorPickerView#67] [General]
Dear @skydoves ,

I am using your colorpicker as ColorPickerDialog version and have a problem to set a default color value that would be shown on build.

Could you be so kind and help me.

Thank you for nice work and kind regards,
Toni

**Comment by skydoves** [comment]
Hi, can you explain more details about your issue?

**Reply by tonipetrovic** [reply]
Hi,

i just want to set the default color (pre-selected color) that will be selected when the dialog appears.

Regards,
Toni
